compiler/rustc_resolve/src/imports.rs

```diff
@@ -667,6 +667,16 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
                     _ => {}
                 }
             }
+
+            match &import.kind {
+                ImportKind::Single { source, .. } => {
+                    if let Some(ModuleOrUniformRoot::Module(module)) = import.imported_module.get()
+                    && let Some(module) = module.opt_def_id() {
```

Comment:
```suggestion
                    if let Some(ModuleOrUniformRoot::Module(module)) = import.imported_module.get()
                      && let Some(module) = module.opt_def_id()
                    {
```
rustfmt cant format let chains yet :(